c10/util/Exception.cpp

```diff
@@ -116,7 +116,7 @@ void torchInternalAssertFail(
 
 } // namespace detail
 
-namespace Warning {
+namespace WarningUtils {
```

Comment:
Not sure if I like the name `WarningUtils`, but I thought that it made sense to name the base warning class `c10::Warning`